pkg/asset/asset.go

```diff
@@ -56,6 +56,9 @@ const (
 	AssetPathSystemNamespace                = "manifests/kube-system-ns.yaml"
 	AssetPathCheckpointer                   = "manifests/pod-checkpointer.yaml"
 	AssetPathEtcdOperator                   = "manifests/etcd-operator.yaml"
+	AssetPathSelfHostedEtcdMemberPeerSecret = "manifests/etcd-member-peer-tls.yaml"
+	AssetPathSelfHostedEtcdMemberCliSecret  = "manifests/etcd-member-client-tls.yaml"
+	AssetPathSelfHostedOperatorEtcdSecret   = "manifests/operator-etcd-client-tls.yaml"
```

Comment:
Sure.
We provide a docs at https://github.com/coreos/etcd-operator/blob/master/doc/user/cluster_tls.md .
Let me know if you don't understand anything in the docs.

> This came up previously when client certs were almost used between etcd peers.

That's the whole point of operator -- expertise written into the code.